cosmo_tester/resources/blueprints/deployment_proxy/infrastructure.yaml

```diff
@@ -0,0 +1,105 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/4.5/types.yaml
+  - plugin:cloudify-openstack-plugin?version=2.0.1
+
+
+inputs:
+  webserver_port:
+    description: The HTTP web server port
+    default: 8080
+  agent_user:
+    description: User name used when SSH-ing into the started machine
+  image:
+    description: Openstack image name or id to use for the new server
+  flavor:
+    description: Openstack flavor name or id to use for the new server
+  network_name:
+    description: Openstack network name the new server will be connected to
+  floating_network_id:
+    description: The id of the network to use for allocating a floating ip
+  key_pair_name:
+    description: Openstack key pair name of the key to associate with the new server
+  private_key_path:
+    description: |
+      Path to the private key which will be used for connecting to the server
+      on the manager or machine running CLI if running in local mode.
+
+
+node_types:
+  cloudify.nodes.ProxyType:
+    derived_from: cloudify.nodes.Root
+    properties:
+      webserver_port:
+        default: { get_input: webserver_port }
+      agent_user:
+        default: { get_input: agent_user }
+      private_key_path:
+        default: { get_input: private_key_path }
+
+node_templates:
+  virtual_ip:
+    type: cloudify.openstack.nodes.FloatingIP
+    properties:
+      floatingip:
+        floating_network_id: { get_input: floating_network_id }
+
+  security_group:
```

Comment:
May don't you put this the app.yaml, so you could remove the use of the port here, and after that, you can change the name to compute_infrastructure or basic_vm or maybe openstack_vm

Reply:
No, that wouldn't work, because I don't want the webapp to be aware of the IaaS, and the security group requires knowledge of OS.

Reply:
In a real-life scenario I maybe would've done something more generic, but for a test, I think it's good enough.